9armbot/player.js

```diff
@@ -74,6 +74,7 @@ class Player{
             coins: coins,
             status: status,
             exp: 0,
+            rollCounter: 0,
```

Comment:
How does this work when player's database has already been established? Will this change introduce bugs? How do we migrate from old version of `Player` to here?

Reply:
Ok i will add migrate now